pynbody/snapshot/ramses.py

```diff
@@ -1046,6 +1048,30 @@ def _not_cosmological(self):
 
         return not_cosmological
 
+    @property
+    def has_negative_iords(self):
+        return self._has_negative_iords()
+
+    def _has_negative_iords(self):
+        return self.dm['iord'].min() < 0
+
+    @property
+    def has_potential_negative_iords_bug(self):
+        return self._has_potential_negative_iords_bug()
+
+    def _has_potential_negative_iords_bug(self):
+        if self.negative_iords_on_purpose:
```

Comment:
should this be `_negative_iords_on_purpose`? Alternatively, if one is going to make it a public member, could the advice to the user offer the possibility of changing this variable rather than reloading their whole file? (thinking of interactive users)